### PR TITLE
[feat] Explore page

### PR DESCRIPTION
**Pull request by Hamdrive**
## Overview

Successfully created explore page for videos.
Now user can easily browse all videos present on the website, either by visiting explore page on navigation, or selecting any particular category

Features:
1. Navigation via navbar or category cards
2. filter videos based on category, addition of reset button
3. Simple cards with thumbnail, title and creator
4. Responsive grid ensures overall clean UI for *almost all* all screen sizes

## Deployment

# https://kemmel-tube-git-page-explore-hamdrive.vercel.app/

## Preview
image
image
image

### CODE REVIEW

File: src/components/navigationWrapper/Navbar.jsx [hunk 1]
```diff
@@ -31,11 +31,32 @@ export const Navbar = () => {
...
+    }
+  }, [location.pathname])
+
   return (
     <nav className="navigation">
       {!(
-        location.pathname.includes("signup") ||
-        location.pathname.includes("login")
+        location?.pathname === ("signup") ||
+        location?.pathname === ("login")
```
Comment (prachi-sahani) [inline src/components/navigationWrapper/Navbar.jsx]: location.pathname will include "/"

File: src/components/navigationWrapper/Topbar.jsx [hunk 2]
```diff
@@ -78,8 +78,8 @@ export const Topbar = ({ setShowSidebar }) => {
   return (
     <div className="topbar">
       {!(
-        location.pathname.includes("signup") ||
-        location.pathname.includes("login")
+        location?.pathname === ("signup") ||
+        location?.pathname === ("login")
```
Comment (prachi-sahani) [inline src/components/navigationWrapper/Topbar.jsx:82]: location.pathname will include "/"

File: src/components/previewCard/PreviewCard.jsx [hunk 3]
```diff
@@ -20,30 +21,33 @@ export const PreviewCard = ({ title, coverImg, description }) => {
...
-          </Typography>
-          <Typography color="#fff" variant="body1">
-            {description}
-          </Typography>
-        </CardContent>
-      </CardActionArea>
+        <CardActionArea
+          component={Link}
+          to="/explore"
+          onClick={onClick}
```
Comment (prachi-sahani) [inline src/components/previewCard/PreviewCard.jsx:27]: better naming convention can be used for onClick function

**Review by prachi-sahani — COMMENTED** [review]
Overall the code is well structured
Good implementation of filters
Good to merge

File: src/components/navigationWrapper/Navbar.jsx (hunk 1 above)
Comment (Hamdrive) [reply src/components/navigationWrapper/Navbar.jsx]: done

File: src/components/navigationWrapper/Topbar.jsx (hunk 2 above)
Comment (Hamdrive) [reply src/components/navigationWrapper/Topbar.jsx:82]: done

File: src/components/previewCard/PreviewCard.jsx (hunk 3 above)
Comment (Hamdrive) [reply src/components/previewCard/PreviewCard.jsx:27]: noted

**Review by shivampandey0 — APPROVED** [review]
- Good use of mui and styled-components.
- Clean code is written.

good to merge

**Comment by Hamdrive** [comment]
Successfully reviewed by 2 team members, will merge to dev and close this branch

**Pull request merged**